documentation/src/main/asciidoc/topics/proc_cli_creating_command_aliases.adoc

```diff
@@ -0,0 +1,33 @@
+[id='cli_creating_command_aliases-{context}']
+= Creating Command Aliases
+Create aliases for {brandname} CLI commands to define custom shortcuts.
+
+.Prerequisites
+
+Create at least one {brandname} user.
```

Comment:
you don't need a user. you can create aliases while disconnected.

Reply:
nice catch. btw you're correct with your question in the comment about *unalias* 'ALIAS-NAME'. `[` and `]` indicate optional args.